src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ShowPlanGraphUtils.cs

```diff
@@ -122,5 +124,26 @@ private static string ParseMissingIndexQueryText(string fileName, string impact,
 */
 ";
         }
+
+        private static string GetPropertyText(PropertyDescriptor property)
+        {
+            try
+            {
+                // Get the property value.
+                object propertyValue = property.GetValue(property);
```

Comment:
Is this correct? Normally you pass in the object that has the property you want the value of. This is getting the value of the descriptor

Reply:
Thanks for pointing it out. Fixed the issue.